lib/components/mobile/batch-search-screen.tsx

```diff
@@ -194,9 +202,11 @@ class BatchSearchScreen extends Component<Props> {
 // connect to the redux store
 
 const mapStateToProps = (state: any) => {
+  const { departArrive } = state.otp.currentQuery
   const currentQuery = state.otp.currentQuery
   return {
-    currentQuery
+    currentQuery,
+    departArrive
```

Comment:
Instead of passing a new prop here, can you extract `departArrive` during render?